angr/concretization_strategies/__init__.py

```diff
@@ -18,29 +20,41 @@ def __init__(self, filter=None, exact=True): #pylint:disable=redefined-builtin
         self._exact = exact
         self._filter = filter
 
+    @staticmethod
+    def _tweak(addr, kwargs):
+        """
+        Utility method used from in here that adds a bogus constraint to extra_constraints making it so that the addr
+        expression can actually be evaluated in all cases
+        """
+        kwargs['extra_constraints'] = kwargs.get('extra_constraints', ()) + (addr == claripy.BVS('TEMP', len(addr)),)
```

Comment:
I would prefer a name more meaningful than `TEMP`.

Reply:
do we really need that? this variable will never be visible to anyone ever except for in debugging, where its purpose should be obvious